lib/engine/game/g_1871/market.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G1871
+      module Market
+        MARKET = [
+          ['', '', '111', '122', '136', '152', '170', '190', '215', '240', '270', '300', '330', '360', '400o'],
+          %w[88 92 100 110 121 133 146 160 180 200 225 250 280],
+          %w[82 86 93 101 111 123 137 152 167 185 203],
+          %w[78 80p 86 94 102 112 122],
+          %w[71 74x 79 84 89 94],
+          %w[60 65z 69 73 78],
+          %w[55 58w 61 65],
+        ].freeze
+
+        # We use par, par_1, par_2, par_3 for our four par options. Use orange
+        # as the 400+40 end square just cause...
```

Comment:
Does the "end game blue" (see 1849 for example) color conflict with the par value colors below?

Reply:
Sorry I don't understand the question. As far as I know the colors I set below are only used for rendering.

1849 denotes the blue part of their market with `u` which is used for phase limiting: https://github.com/tobymao/18xx/blob/1233d5f60b6973554cc2331de1ec2282c19f0304/lib/engine/share_price.rb#L26

I use the `p,x,z,w` options which are `par,par_1,par_2,par_3`. I didn't have to edit base game code for that. Only set their description strings below and remove those pars on phase events (also doable without editing base game code).